taskcluster/ci/toolchain/android.yml

```diff
@@ -45,9 +45,6 @@ linux64-android-gradle-dependencies:
             - taskcluster/scripts/toolchain/android-gradle-dependencies.sh
             - taskcluster/scripts/toolchain/android-gradle-dependencies/**
             - buildSrc/src/main/java/Dependencies.kt
-            - '*.gradle'
-            - components/**/build.gradle
-            - samples/**/build.gradle
```

Comment:
We should keep the root `build.gradle` file since that contains some plugin versions!

Reply:
Good point! Instead of the `*.gradle` then we can use the file directly as `build.gradle` so we don't watch other files unnecessarily.